pkg/server/chaosd/runtime_client.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Chaos Mesh Authors.
```

Comment:
You can use this pkg directly! https://github.com/chaos-mesh/chaosd/blob/main/pkg/crclient/client.go

Reply:
removed